### ISSUE TITLE
HTML Tag acronym causes line break

### ISSUE THREAD

**Issue by karvulf** [archriss/react-native-render-html#251]
### Is this a bug report or a feature request?

Feature Request

### Have you read [the documentation](https://github.com/archriss/react-native-render-html/blob/master/README.md) in its entirety?

Yes

### Have you made sure that your issue hasn't already been reported/solved?

Yes

### Is the bug specific to iOS or Android? Or can it be reproduced on both platforms?

Its a problem in ios and android.

### Is the bug reproductible in a production environment (not a debug one)?

Yes.

### Steps to Reproduce

1. Give as html: 'Schwingen <acronym title="Neutrum">nt</acronym>' in the html-Prop.
2. After rendering, the content in acronym gets a line break.

Example:
```
<HTML
    html={'Schwingen <acronym title="Neutrum">nt</acronym>'}
/>
```
### Expected Behavior

There shouldn't be a line break.

### Actual Behavior

The name "nt" appears in the next line.

**Comment by jsamr** [comment]
acronym is [not yet supported](https://github.com/archriss/react-native-render-html/blob/c1178fdb20ceb404d9a843a9e273bd6ca6bbb3c1/src/HTMLUtils.js#L16). We would just need to add `acronym` to the list of TEXT_TAGS so that they are rendered inline.

**Comment by jsamr** [comment]
This issue will be fixed in version 4.2.2.
A pre-release is already available if you want this fix now, see #385.
